proj/cmake/libcinder_target.cmake

```diff
@@ -24,11 +24,12 @@ target_include_directories( cinder SYSTEM BEFORE INTERFACE ${CINDER_INCLUDE_SYST
 target_include_directories( cinder BEFORE PRIVATE ${CINDER_INCLUDE_USER_PRIVATE} )
 target_include_directories( cinder SYSTEM BEFORE PRIVATE ${CINDER_INCLUDE_SYSTEM_PRIVATE} )
 
-target_link_libraries( cinder PUBLIC ${CINDER_LIBS_DEPENDS} )
+target_link_libraries( cinder PUBLIC ${CINDER_LIBS_DEPENDS}  )
+target_link_libraries( cinder LINK_PRIVATE Ws2_32.lib  )
```

Comment:
Sorry, that was apart of my experimenting to find some better way to propagate the link flags. It's been removed now.